build-setup/linux.md

```diff
@@ -237,6 +237,67 @@ sudo make install
 
 to complete the build.
 
+### CMake
+
+CMake version 3.5 is the minimum required to build PyTorch. Download version 3.19.3 from <https://github.com/Kitware/CMake/releases/download/v3.19.3/cmake-3.19.3-Linux-x86_64.sh> and install:
+
+```
+chmod +x cmake-3.19.3-Linux-x86_64.sh
+mkdir /usr/local/cmake
+./cmake-3.19.3-Linux-x86_64.sh --skip-license --prefix=/usr/local/cmake
```

Comment:
```suggestion
sudo mkdir /usr/local/cmake
sudo ./cmake-3.19.3-Linux-x86_64.sh --skip-license --prefix=/usr/local/cmake
```

The Dockerfile runs as root, whereas the average user with a Linux laptop won't be, so best to include the `sudo` in these instructions.